testSuite/scripts/utility.py

```diff
@@ -566,7 +566,7 @@ def verify_operation(command):
             command, stderr=subprocess.STDOUT, shell=True, timeout=600,
             universal_newlines=True)
     except subprocess.CalledProcessError as exec:
-        print("command failed with error code ", exec.returncode, " and message " + exec.output)
+        # print("command failed with error code ", exec.returncode, " and message " + exec.output)
```

Comment:
uncomment?

Reply:
This is an inconsistent uncomment if we do. Furthermore, we need to sanitize the test framework properly as I've seen SAS tokens leak out.